src/main/java/hudson/plugins/gradle/GradleConsoleAnnotator.java

```diff
@@ -28,7 +33,12 @@ protected void processLogLine(String line) throws IOException {
         line = trimEOL(line);
 
         if (annotateGradleOutput) {
-            if (line.startsWith(":") || line.startsWith("> Task :")) { // put the annotation
+            // assumption that the timestamp prefix will be present or not for all lines
+            if (timestampPrefix == null) {
+                timestampPrefix = detectTimestampPrefix(line);
+            }
```

Comment:
Can it be the case that one line has the timestamp and the next one doesn't?

Reply:
From what I saw, it should not be the case. I think it's ok to take this risk/assumption vs detecting the timestamp on every log line.